curriculum/challenges/english/25-front-end-development/quiz-testing/66f1aeffc5774ac692112a7e.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What type of testing focuses on verifying individual components or functions of code in isolation?
 
 #### --distractors--
 
-Placeholder distractor 1
+'Functional testing'
```

Comment:
```suggestion
Functional testing
```
Quotes are not needed around distractors or answer. Please remove them for all cases.